app/controllers/courses_controller.rb

```diff
@@ -122,6 +122,12 @@ def alerts
     @alerts = current_user&.admin? ? @course.alerts : @course.public_alerts
   end
 
+  def suspected_plagiarism
+    set_course
+    @revisions = @course.suspected_plagiarism
+    @show_report = current_user&.admin?
```

Comment:
This should be available to both admins and course instructors. You can do that with `current_user&.can_edit?(@course)`; if a user is allowed to edit a course, it they are either an admin or an instructor for the particular course.